index.html

```diff
@@ -0,0 +1,166 @@
+</head>
+<body id="body">
+    <main>
+        <div class="socials">
+            <ul class="social-m">
+                <li><a href="#" class="nono facebook-to"><i class="fab fa-facebook-f"></i></a></li>
+                <li><a href="#" class="nono twitter-to"><i class="fab fa-twitter"></i></a></li>
+                <li><a href="#" class="nono"><p class="fab">English</p></a></li>
+                <li><a href="#" class="nono"><p class="fab">Log out</p></a></li>
+            </ul>
+        </div>
+        <nav class="nav-bar">
+            <span class="btn humberger-btn"><i class="fas fa-bars" onclick="changeIcon(this)"></i></span>
+            <label class="logo"><a href="index.html" class="refr"><img src="IMAGES/darko-logo.png" alt="logo" class="logo-desk"></a></label>
+            <ul class="menu">
+                <li class="menu-choice home"><a href="index.html" class="link-desk">Home</a></li>
+                <li class="menu-choice about"><a href="About.html" class="link-desk">About</a></li>
+                <li class="menu-choice blog"><a href="#" class="link-desk">Blog</a></li>
+                <li class="menu-choice news"><a href="#" class="link-desk">News</a></li>
+                <li class="menu-choice sponsor"><a href="#" class="link-desk">Sponsor</a></li>
+                <li class="menu-choice call" id="contact"><a href="#" class="link-desk link-desko">Call Us</a></li>
+            </ul>
+        </nav>
+            <ul class="options">
+                <li class="choice"><a href="index.html" class="go-to-page">Home</a></li>
+                <li class="choice "><a href="About.html" class="go-to-page">About</a></li>
+                <li class="choice"><a href="#" class="go-to-page">Blog</a></li>
+                <li class="choice"><a href="#" class="go-to-page">Sponsor</a></li>
+                <li class="choice"><a href="#" class="go-to-page hh">News</a></li>
+                <li class="choice"><a href="#" class="go-to-page">Contact</a></li>
+            </ul>
+        <section class="introduction">
+            <p class="intro greeting">Hello! Future Musicians,</p>
+            <h1 class="intro title">DARKO MUSIC COURSE 2022 SUMMER INTAKE</h1>
+            <p class="intro description">Darko Music is glad to inform you that the application for 2022 Summer Class is open to those who may be willing to join the community.</p>
+            <h2 class="intro time">2022.02.05(SAT) - 2022.04.05(TUE)</h2>
+            <p class="intro tag">Darko Music, Rwanda, Kigali</p>
+        </section>
+        <!-- MAIN PROGRAMS-->
+        <section class="main-programs">
+            <div class="title"><h2 class="header">Main Program</h2></div>
+            <div class="underline"></div>
+            <div class="programs">
+                <!-- PROGRAM 1 -->
+                <div class="program lecturing">
+                    <div class="program-head">
+                        <img src="IMAGES/competition.png" alt="Lecturing" class="p-image">
```

Comment:
- Well done on adding the main program section but there are some issues that you have to fix.
- [ ] First I suggest you move the underline under Main Program more to the center following the design
- [ ] Good job on adding the background but kindly use the same background as it was given in the design. You can use this image: 
![repeat-bg](https://user-images.githubusercontent.com/74506933/163674090-1e311830-e032-40c7-8334-91bb8fb1390a.png) and add background-repeat: repeat; and it will look perfect.
- [ ] Good job on styling the cards but please follow the [design guidlines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) styling the text and everything else. Keep coding 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/163674146-41c570cb-ac5a-49ff-9b44-d6083cffa681.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/163674151-5a278199-58f5-4814-b9d0-e5c16d6b67a6.png)